content/docs/04-clusters/04-palette-virtual-clusters/palette-virtual-cluster-quickstart.md

```diff
@@ -59,24 +67,29 @@ You can enable Palette Virtual Clusters on an existing host cluster by performin
 1. From the slide menu, select **Clusters** and view the list of **Clusters**.
 
 
-2. Click any Host Cluster from the list and select **Settings** > **Cluster Settings** > **Palette Virtual Clusters**.
-
+2. Click any Host Cluster from the list and select **Settings** > **Cluster Settings** > **Virtual Clusters**.
 
-3. Toggle the **Enable Palette Virtual Clusters** option (yes/no).
 
-    **Note**: This feature can be enabled when first creating a new cluster, so you may find that it is already enabled.
+3. Toggle the **Enable Virtual Clusters** option (yes/no).
+<br />
```

Comment:
@ritawatson - looks like this page was wrote probably when we only supported direct deployment on host clusters. I think deploying via the Cluster Group option can also be part of the same document.